Doc/source/conf.py

```diff
@@ -137,8 +137,8 @@
 
 # The name for this set of Sphinx documents.  If None, it defaults to
 # "<project> v<release> documentation".
-html_title = "%s v%s Manual (DRAFT)" % (project, version)
-#html_title = "%s v%s Manual" % (project, version)
+#html_title = "%s v%s Manual (DRAFT)" % (project, version)
+html_title = "%s v%s Manual" % (project, version)
```

Comment:
Should this be changed back in the 0.4.1a0 commit?

Reply:
Yes, it should.